<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QDialog" name="SettingDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SettingDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okBtn">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="settingsTab">
      <attribute name="title">
       <string>设置</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QGroupBox" name="commonGroupBox">
         <property name="title">
          <string>常规</string>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QCheckBox" name="appAutoRunCheckBox">
            <property name="text">
             <string>开机自动启动</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="verboseLogCheckBox">
            <property name="text">
             <string>记录详细日志</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>151</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="advancedTab">
      <attribute name="title">
       <string>高级</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0" colspan="2">
        <widget class="QGroupBox" name="imageProcessGroupBox">
         <property name="title">
          <string>Image Process</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QCheckBox" name="sensorCheckBox">
            <property name="text">
             <string>Sensor triggered</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="angleHighLabel">
                <property name="text">
                 <string>highRatio</string>
                </property>
                <property name="buddy">
                 <cstring>angleHighSpinBox</cstring>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="angleHighSpinBox">
                <property name="minimum">
                 <double>1.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>10.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.010000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="angleLowLabel">
                <property name="text">
                 <string>lowRatio</string>
                </property>
                <property name="buddy">
                 <cstring>angleLowSpinBox</cstring>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="angleLowSpinBox">
                <property name="maximum">
                 <double>1.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.010000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="radiusMinLabel">
                <property name="text">
                 <string>&amp;radius_min</string>
                </property>
                <property name="buddy">
                 <cstring>radiusMinSpinBox</cstring>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_6">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QSpinBox" name="radiusMinSpinBox">
                <property name="maximum">
                 <number>1000</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="radiusMaxLabel">
                <property name="text">
                 <string>radius_max</string>
                </property>
                <property name="buddy">
                 <cstring>radiusMaxSpinBox</cstring>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_8">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QSpinBox" name="radiusMaxSpinBox">
                <property name="maximum">
                 <number>1000</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLabel" name="roiLabel_1">
                <property name="text">
                 <string>roiRect(</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="roiSpinBox_x">
                <property name="maximum">
                 <number>1280</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="roiLabel_2">
                <property name="text">
                 <string>,</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="roiSpinBox_y">
                <property name="maximum">
                 <number>720</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="roiLabel_3">
                <property name="text">
                 <string>,</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="roiSpinBox_w">
                <property name="maximum">
                 <number>1280</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="roiLabel_4">
                <property name="text">
                 <string>,</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="roiSpinBox_h">
                <property name="maximum">
                 <number>720</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="roiLabel_5">
                <property name="text">
                 <string>)</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="videoCaptureGroupBox">
         <property name="title">
          <string>Video Capture</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="capIntervalLabel">
              <property name="text">
               <string>&amp;capInterval</string>
              </property>
              <property name="buddy">
               <cstring>capIntervalSpinBox</cstring>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_9">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QSpinBox" name="capIntervalSpinBox">
              <property name="maximum">
               <number>1000</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>sensorCheckBox</tabstop>
  <tabstop>angleHighSpinBox</tabstop>
  <tabstop>angleLowSpinBox</tabstop>
  <tabstop>radiusMinSpinBox</tabstop>
  <tabstop>radiusMaxSpinBox</tabstop>
  <tabstop>capIntervalSpinBox</tabstop>
  <tabstop>okBtn</tabstop>
  <tabstop>cancelBtn</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>okBtn</sender>
   <signal>clicked()</signal>
   <receiver>SettingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>533</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>SettingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>440</x>
     <y>533</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
